app/lib/provider/persistence_provider.dart

```diff
@@ -82,6 +82,7 @@ const _enableAnimations = 'ls_enable_animations';
 const _deviceType = 'ls_device_type';
 const _deviceModel = 'ls_device_model';
 const _shareViaLinkAutoAccept = 'ls_share_via_link_auto_accept';
+const _privacyProtectionMode = 'privacy_protection_mode';
```

Comment:
Please change it to `ls_share_media_metadata`

Reply:
changed